### PR TITLE
[RFC] | Title - Reflect the page that you're currently on

### PR DESCRIPTION
**Pull request by alexandru-patrascu**
Task from Jira: https://wingravity.atlassian.net/browse/OC-77

### CODE REVIEW

File: views/partials/header.hbs [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
     {{#if publication}}
-     Publication: {{publication.title}} - Octopus
-    {{else}}
-      {{#if problem}}
-      Problem: {{problem.title}} - Octopus
-      {{else}}
-       Octopus. Built for Scientists.
-     {{/if}}
-   {{/if}}
+    Publication: {{publication.title}} - Octopus
```
Comment (vot) [inline views/partials/header.hbs]: Can we turn this into a simpler server-side assignment?

Each controller can simply assign / overwrite `res.locals.customTitleTag` to provide the requested title as a string.

Then in template we can simplify this like this:
```
{{#if customTitleTag}}
      {{customTitleTag}} - Octopus
{{else}}
       Octopus. Built for Scientists.
{{/if}}
```

**Review by vot — CHANGES_REQUESTED** [review]

File: views/partials/header.hbs (hunk 1 above)
Comment (alexandru-patrascu) [reply views/partials/header.hbs]: @vot Done

File: views/partials/header.hbs [hunk 2]
```diff
@@ -1,19 +1,13 @@
...
-    {{#if publication}}
-     Publication: {{publication.title}} - Octopus
-    {{else}}
-      {{#if problem}}
-      Problem: {{problem.title}} - Octopus
-      {{else}}
-       Octopus. Built for Scientists.
-     {{/if}}
-   {{/if}}
+    {{customTitleTag}}
```
Comment (vot) [inline views/partials/header.hbs:9]: Can we still have the else condition with a default fallback?

Comment (alexandru-patrascu) [reply views/partials/header.hbs:9]: @vot the default fallback is assigned server-side in **routes/middleware.js** line 26

**Review by danielberigoi — APPROVED** [review]

**Review by vot — APPROVED** [review]
Looks good, very readable and easy to follow the context 👍

**Pull request merged**